torch/csrc/api/src/nn/modules/dropout.cpp

```diff
@@ -45,5 +46,16 @@ Tensor FeatureDropoutImpl::forward(const Tensor& input) {
 void FeatureDropoutImpl::pretty_print(std::ostream& stream) const {
   stream << "torch::nn::FeatureDropout(rate=" << options.rate() << ")";
 }
+
+AlphaDropoutImpl::AlphaDropoutImpl(const DropoutOptions& options_)
+    : DropoutImplBase(options_) {}
+
+Tensor AlphaDropoutImpl::forward(const Tensor& input) {
+  return torch::alpha_dropout(input, options.rate(), this->is_training());
+}
+
+void AlphaDropoutImpl::pretty_print(std::ostream& stream) const {
+  stream << "torch::nn::AlphaDropout(rate=" << options.rate() << ")";
```

Comment:
We might want to print both `p` and `inplace`, to match the Python version even better :D